doc-Migration_Analytics_Guide/cfme/modules/Creating-migration-analytics-report.adoc

```diff
@@ -20,12 +20,12 @@ The *Reports* window is displayed.
 . Click *Create* and fill in the following fields:
 
 * Mandatory fields:
-** *Inventory data file*: Inventory data file generated by CloudForms. Click *Browse* to select the `tar.gz` file.
-** *Report name*: Name of your report, up to 250 characters
-** *Year-over-year growth rate for new hypervisors*: Number of new hypervisors to be added each year
+** *Inventory data file*: `tar.gz` archive file containing one or more `.json` inventory data files. Click *Browse* to select the `tar.gz` file.
```

Comment:
Forgot to fix this. I guess a preview would have been a good idea after all.

`tar.gz` archive file containing one or more `.json` inventory data files. Click *Browse* to select the `tar.gz` file.

Please remove ``tar.gz` archive file containing one or more `.json` inventory data files.`